modules/iam/src/main/scala/ch/epfl/bluebrain/nexus/commons/iam/IamClient.scala

```diff
@@ -38,19 +41,20 @@ trait IamClient[F[_]] {
 }
 
 object IamClient {
-  private val log  = Logger[this.type]
-  private val Acls = Path("acls")
-  private val User = Path("oauth2/user")
+  private val log               = Logger[this.type]
+  private val Acls              = Path("acls")
+  private val User              = Path("oauth2/user")
+  private val filterGroupsParam = "filterGroups"
 
   final def apply()(implicit ec: ExecutionContext,
                     aclClient: HttpClient[Future, AccessControlList],
                     userClient: HttpClient[Future, User],
                     iamUri: IamUri): IamClient[Future] = new IamClient[Future] {
 
-    override def getCaller(optCredentials: Option[OAuth2BearerToken]) =
+    override def getCaller(optCredentials: Option[OAuth2BearerToken], filterGroups: Boolean = false) =
```

Comment:
I think the default value should actually be true, cause that's what we are gonna end up using always

Reply:
I think `false` is better default value
#33 
> I would say false would be a better default as it would be backward compatible and also less surprising when you call getCaller with defaults

Reply:
In this case, in order for this to be finish you'll have to deploy, upgrade the version of `commons` in kg and add the `.getCaller(filterGroups  = true)`.